test/doctype.html

```diff
@@ -0,0 +1,50 @@
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>doctype test</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+  <link rel="import" href="../../test-fixture/test-fixture.html">
+
+  <link rel="import" href="../../polymer/polymer-element.html">
+  <link rel="import" href="../vaadin-element-mixin.html">
+</head>
+
+<body>
+
+  <test-fixture id="default">
+    <template>
+      <x-element></x-element>
+    </template>
+  </test-fixture>
+
+  <script>
+    window.addEventListener('WebComponentsReady', () => {
+      customElements.define('x-element', class extends Vaadin.ElementMixin(Polymer.Element) {
+        static get is() {
+          return 'x-element';
+        }
+      });
+    });
+
+    describe('missing doctype', () => {
+      beforeEach(done => {
+        customElements.whenDefined('x-element').then(done);
+      });
+
+      it('should warn about doctype', () => {
+        const _warn = console.warn;
+        const spy = console.warn = sinon.spy();
+        fixture('default');
+        console.warn = _warn;
+        expect(spy.called).to.be.true;
```

Comment:
Would it be better to use `calledWith` (https://sinonjs.org/releases/v2.0.0/assertions/) and verify that the warning message is correct?

E.g. https://github.com/sweetalert2/sweetalert2/blob/13662349768bfdd98cfb91000e55bf2c3cfc3bb9/test/qunit/params/type.js#L11

Reply:
O-ho, I'm apparently too slow :)